controllers/admins.js

```diff
@@ -0,0 +1,104 @@
+const admins = require('../data/admins.json');
+const fs = require('fs');
+
+const getAllAdmins = (req, res) => {
+    res.send(admins);
+};
+
+const getAdminById = (req, res) => {
+    const founded = admins.some(admin => {
+        return admin.id === parseInt(req.params.id);
+    });
+    if (founded) {
+        res.status(200).json(admins.filter(admin => admin.id === parseInt(req.params.id)));
+    } else {
+        res.status(400).json({ msg: `There is no admin with this id: ${req.params.id}` });
```

Comment:
The status code for when the requested resource hasn't been found is `404`.

Also, there's no need to return a message as a json format inside of an object. We could just a string instead:

```suggestion
        res.status(404).send(`There is no admin with this id: ${req.params.id}`);
```